src/models/Transaction/index.ts

```diff
@@ -10,8 +10,8 @@ export interface CellDep {
 
 export interface Transaction {
   transactionHash: string
-  blockNumber: number
-  blockTimestamp: number
+  blockNumber: number | string
+  blockTimestamp: number | string
```

Comment:
This is our internal type definition in the frontend and should not be affected by incorrect external types.

External data should be converted to the correct internal model when entering the internal layer through the service layer.

Reply:
Indeed, but I think the complementation of `string` is more advisable for this case because the type `Transaction` is directly used to describe the response of API without a transformation.

But the existent API returns `string` instead of `number` here, so the type declaration is incorrect.

The proper solution is to introduce a transformation between the response and the internal state, but it will introduce too many irrelevant changes in this PR. So it should be done in an independent PR.

Ref:
Response described by `Transaction`: https://github.com/Magickbase/ckb-explorer-frontend/blob/develop/src/services/ExplorerService/fetcher.ts#L73
`block_number` and `block_timestamp` in response: 
<img width="402" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/7271329/db5bbb93-db90-4edb-b624-827ca6d9d8a6">

Reply:
> Indeed, but I think the complementation of `string` is more advisable for this case because the type `Transaction` is directly used to describe the response of API without a transformation.

In terms of abstraction, the Transaction in the model is not intended to describe the structure returned by the API. We can consider it as coincidentally matching the structure returned by the API.

> The proper solution is to introduce a transformation between the response and the internal state, but it will introduce too many irrelevant changes in this PR(maybe going through all APIs). So it should be done in an independent PR.

Indeed, I think we can add a TODO comment here to explain the source of the string type, to avoid forgetting which places need adjustment when considering future refactoring. Testing all the interfaces comprehensively would be a massive task at that time.

Reply:
Comment was added by https://github.com/Magickbase/ckb-explorer-frontend/pull/175/commits/d56acfb29a1d877a45d95d39dea53bc1a7a4e91f